### PR TITLE
We can delete from the database!

### PR DESCRIPTION
**Pull request by afontaine**
There's a few other changes, too. Please take a look. I'll be writing tests on
this branch, too. Just take a look and point it out. 

I added some stuff in StoryManager, not entirely sure if it's right, or if more
is needed.

### CODE REVIEW

File: adventure.datetime/src/ca/cmput301f13t03/adventure_datetime/model/StoryManager.java [hunk 1]
```diff
@@ -241,8 +241,7 @@ public boolean putStory(Story story) {
 	 * Delete a story from the database
 	 */
 	public void deleteStory(String storyId) {
-		// TODO Needs to be implemented in database.
-
+		m_db.deleteStory(storyId);
```
Comment (edegraff) [inline adventure.datetime/src/ca/cmput301f13t03/adventure_datetime/model/StoryManager.java:277]: Remove from local cache.
Publish.

File: adventure.datetime/src/ca/cmput301f13t03/adventure_datetime/model/StoryManager.java [hunk 2]
```diff
@@ -277,8 +276,7 @@ public boolean putFragment(StoryFragment fragment) {
 	 * Delete a fragment from the database
 	 */
 	public void deleteFragment(UUID fragmentId) {
-		// TODO Needs to be implemented in database.
-
+		m_db.deleteStoryFragment(fragmentId.toString());
```
Comment (edegraff) [inline adventure.datetime/src/ca/cmput301f13t03/adventure_datetime/model/StoryManager.java:317]: Remove from local cache.
Publish.

File: adventure.datetime/src/ca/cmput301f13t03/adventure_datetime/model/StoryManager.java [hunk 3]
```diff
@@ -263,11 +297,14 @@ public Story getStory(String storyId) {
 	 */
 	public boolean putFragment(StoryFragment fragment) {
```
Comment (jesseDtucker) [inline adventure.datetime/src/ca/cmput301f13t03/adventure_datetime/model/StoryManager.java:299]: whitespace...?

**Comment by jesseDtucker** [comment]
Approved, Test the deletes a bit if you can. I'm slightly worried I missed a bug in the cache updates but I believe it should be fine.

File: adventure.datetime/src/ca/cmput301f13t03/adventure_datetime/model/StoryManager.java (hunk 3 above)
Comment (afontaine) [inline adventure.datetime/src/ca/cmput301f13t03/adventure_datetime/model/StoryManager.java:299]: I guess so? Probably added in when I merged a bunch of stuff.

**Comment by jviau** [comment]
Any chance you would want to change the String IDs to UUIDs, just for this new delete stuff. I just refactored all the current stuff to use UUID's. Or if you leave it in I will quickly change it over the do a pull request for my stuff.

**Comment by edegraff** [comment]
Tested and approved.

**Comment by jesseDtucker** [comment]
Are we pulling this in or not?

**Comment by afontaine** [comment]
Yes.

**Pull request merged**